modules/wmco-prerequisites.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/understanding-windows-container-workloads.adoc
+
+[id="wmco-prerequisites_{context}"]
+= Windows Machine Config Operator (WMCO) prerequisites
+
+The following information details the supported cloud provider versions, Windows Server versions, and networking configurations. See the vSphere documentation for any information that is relevant to only that platform.
+
+== Supported Cloud Providers based on OCP and WMCO versions
+
+[cols="3",options="header"]
+|===
+| Cloud Provider           | Supported OKD/OCP Version   | Supported WMCO version
+|Amazon Web Services (AWS) |4.6+ |WMCO 1.0+
+|Azure |4.6+ |WMCO 1.0+
```

Comment:
My temptation is to suggest "Microsoft Azure", but I'm having difficulty finding a guideline.